js_modules/dagit/packages/core/src/assets/AutoMaterializePolicyPage/AssetAutomaterializePolicyPage.tsx

```diff
@@ -375,16 +387,16 @@ const MiddlePanel = ({
   maxMaterializationsPerMinute,
 }: {
   assetKey: Omit<AssetKey, '__typename'>;
-  selectedEvaluationId?: string;
+  selectedEvaluationId?: number;
   maxMaterializationsPerMinute: number;
 }) => {
   const {data, loading, error} = useQuery<GetEvaluationsQuery, GetEvaluationsQueryVariables>(
     GET_EVALUATIONS_QUERY,
     {
       variables: {
         assetKey,
-        cursor: selectedEvaluationId,
-        limit: 1,
+        cursor: selectedEvaluationId ? (selectedEvaluationId + 1).toString() : undefined,
```

Comment:
[The +1 here is awkward](https://elementl-workspace.slack.com/archives/C04UD72HHQX/p1685129625555949?thread_ts=1685126824.627239&cid=C04UD72HHQX)